mysqld_exporter.go

```diff
@@ -59,11 +54,21 @@ var (
 		"config.my-cnf",
 		"Path to .my.cnf file to read MySQL credentials from.",
 	).Default(path.Join(os.Getenv("HOME"), ".my.cnf")).String()
+	mysqldHost = kingpin.Flag(
+		"mysqld.host",
+		"Hostname to use for connecting to MySQL",
+	).Default("localhost").String()
```

Comment:
Shouldn't this be `mysql.address`? With a default of `localhost:3306`? 

Then `ReloadConfig()` can use `net.SplitHostPort()` to get the host and port.

Reply:
Good point, for some reason I was thinking that I had added a `port` flag as well.

Decided against `net.SplitHostPort()` since it doesn't allow port to be optional, which would cause errors if a user specifies the `mysqld.address` without a port.